iphone/cli/commands/_build.js

```diff
@@ -207,6 +207,34 @@ iOSBuilder.prototype.assertIssue = function assertIssue(issues, name) {
 };
 
 /**
+ * Retreieves the certificate information by name.
+ *
+ * @param {String} name - The cert name.
+ * @param {String} [type] - The type of cert to scan (developer or distribution).
+ * @returns {Object|null}
+ * @access private
+ */
+iOSBuilder.prototype.getCert = function getCert(name, type) {
+	if (name && this.iosInfo) {
+		for (const keychain of Object.keys(this.iosInfo.certs.keychains)) {
+			const scopes = this.iosInfo.certs.keychains[keychain];
+			const types = type ? [ type ] : Object.keys(scopes);
+			for (const scope of types) {
+				if (scopes[scope]) {
+					for (const cert of scopes[scope]) {
+						if (cert.name === name) {
```

Comment:
Jenkins: `Blocks are nested too deeply (6)  max-depth`. May want to use `Array.prototype.find()` for the last check? Otherwise, move into different parts or increase the nested-limit.

Reply:
I chose to suppress the eslint exception. It's a short function and despite having deep code, it's all single line statements.